src/components/TodoModal/TodoModal.tsx

```diff
@@ -2,11 +1,38 @@
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+
+interface Props {
+  onResetSelectedTodo: () => void;
+  selectedTodo: Todo;
+}
+
+export const TodoModal: React.FC<Props> = ({
+  onResetSelectedTodo,
+  selectedTodo,
+}) => {
+  const [loadingTodoModal, setLoadingTodoModal] = useState(false);
+  const [user, setUser] = useState<User | null>(null);
+
+  useEffect(() => {
+    setLoadingTodoModal(true);
+    getUser(selectedTodo.userId)
+      .then(setUser)
+      .finally(() => setLoadingTodoModal(false));
+  }, [selectedTodo.userId]);
+
+  const handleResetUserAndTodo = () => {
+    setUser(null);
+    onResetSelectedTodo();
+  };
 
-export const TodoModal: React.FC = () => {
   return (
```

Comment:
You are interacting with the DOM directly here by setting the class name. This is against the react way of doing things. Instead, you should use the classnames library to conditionally set the class name based on the state or props.